docs/src/pages/demos/badges/BadgeVisibility.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, { useState } from 'react';
```

Comment:
There are no dead code elimination capabilities with React, let's ease the DX by using `React.`

```diff
-import React, { useState } from 'react';
+import React from 'react';
```